bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/driver/Port.java

```diff
@@ -83,6 +83,7 @@
     private ModemDBBuilder mdbb;
     private ArrayList<MsgListener> listeners = new ArrayList<>();
     private LinkedBlockingQueue<Msg> writeQueue = new LinkedBlockingQueue<>();
+    private Boolean disconnected = false;
```

Comment:
If you aren't going to change this to an `AtomicBoolean` then please make this primitive.

Reply:
Changed to AtomicBoolean